benchmark_results_data/evm_benchmarks.csv

```diff
@@ -1,101 +1,108 @@
-cita-evm,bn128_mul_weierstrudel-chfast1,0.007357496,49441
-evmone,bn128_mul_weierstrudel-chfast2,0.000458,39.337k
-parity-evm,bn128_mul_weierstrudel-chfast2,0.003425564,39337
-geth-evm,bn128_mul_weierstrudel-chfast2,0.001819083,39337
-cita-evm,bn128_mul_weierstrudel-chfast2,0.014044644,66865
-evmone,bn128_mul_weierstrudel-chfast3,0.000525,47.291k
-parity-evm,bn128_mul_weierstrudel-chfast3,0.004222961,47291
-geth-evm,bn128_mul_weierstrudel-chfast3,0.002231105,47291
-cita-evm,bn128_mul_weierstrudel-chfast3,0.017467266,74819
-evmone,bn128_mul_weierstrudel-cdetrio6,0.000541,47.561k
-parity-evm,bn128_mul_weierstrudel-cdetrio6,0.004162919,47561
-geth-evm,bn128_mul_weierstrudel-cdetrio6,0.002232338,47561
-cita-evm,bn128_mul_weierstrudel-cdetrio6,0.017366266,75089
-evmone,bn128_mul_weierstrudel-cdetrio7,0.000523,47.165k
-parity-evm,bn128_mul_weierstrudel-cdetrio7,0.00429027,47165
-geth-evm,bn128_mul_weierstrudel-cdetrio7,0.002412266,47165
-cita-evm,bn128_mul_weierstrudel-cdetrio7,0.018339345,74501
-evmone,bn128_mul_weierstrudel-cdetrio11,0.000513,47.561k
-parity-evm,bn128_mul_weierstrudel-cdetrio11,0.004136138,47561
-geth-evm,bn128_mul_weierstrudel-cdetrio11,0.00215742,47561
-cita-evm,bn128_mul_weierstrudel-cdetrio11,0.016865172,75089
-evmone,bn128_mul_weierstrudel-cdetrio12,0.000571,47.165k
-parity-evm,bn128_mul_weierstrudel-cdetrio12,0.004018633,47165
-geth-evm,bn128_mul_weierstrudel-cdetrio12,0.002310364,47165
-cita-evm,bn128_mul_weierstrudel-cdetrio12,0.018379059,74501
-evmone,blake2b_huff-8415-bytes,0.002448,879.92k
-parity-evm,blake2b_huff-8415-bytes,0.027908764,879920
-geth-evm,blake2b_huff-8415-bytes,0.035075896,879920
-cita-evm,blake2b_huff-8415-bytes,0.015139534,1471220
-evmone,blake2b_huff-5610-bytes,0.001544,586.839k
-parity-evm,blake2b_huff-5610-bytes,0.017326637,586839
-geth-evm,blake2b_huff-5610-bytes,0.023309606,586839
-cita-evm,blake2b_huff-5610-bytes,0.010120834,988103
-evmone,blake2b_huff-2805-bytes,0.000844,293.755k
-parity-evm,blake2b_huff-2805-bytes,0.008337344,293755
-geth-evm,blake2b_huff-2805-bytes,0.011073158,293755
-cita-evm,blake2b_huff-2805-bytes,0.005137996,504983
-evmone,sha1-10808-bits,0.003175,790.156k
-parity-evm,sha1-10808-bits,0.055919944,790156
-geth-evm,sha1-10808-bits,0.020203397,790156
-cita-evm,sha1-10808-bits,0.053636084,903488
-evmone,sha1-21896-bits,0.005928,1.54361M
-parity-evm,sha1-21896-bits,0.120435923,1543605
-geth-evm,sha1-21896-bits,0.0451866,1543605
-cita-evm,sha1-21896-bits,0.10864109,1750413
-evmone,sha1-42488-bits,0.01191,3.0144M
-parity-evm,sha1-42488-bits,0.223914963,3014404
-geth-evm,sha1-42488-bits,0.082095587,3014404
-cita-evm,sha1-42488-bits,0.203935243,3396188
-evmone,mul256-gcolvin-drag-race,0.011393,5.61028M
-parity-evm,mul256-gcolvin-drag-race,0.114632564,5610277
-geth-evm,mul256-gcolvin-drag-race,0.167468693,5610277
-cita-evm,mul256-gcolvin-drag-race,0.043008777,5635901
+engine, test_name, total_time, gas_used
+engine,test_name,total_time,gas_used
```

Comment:
One of these lines seem to be stray

Reply:
annoying to see, but doesn't seem to break the notebook.